common-lib/src/main/scala/com/gu/mediaservice/lib/config/Services.scala

```diff
@@ -30,28 +30,9 @@ class Services(val domainRoot: String, isProd: Boolean, hosts: ServiceHosts) {
 
   val guardianWitnessBaseUri: String = "https://n0ticeapis.com"
 
-  val toolsDomains: Set[String] = if(isProd) {
-    Set(domainRoot)
-  } else {
-    Set(
-      domainRoot.replace("test", "local"),
-      domainRoot.replace("test", "code")
-    )
-  }
-
-  // TODO move to config
-  val corsAllowedTools: Set[String] = toolsDomains.foldLeft(Set[String]()) {(acc, domain) => {
-    val tools = Set(
-      baseUri(s"composer.$domain"),
-      baseUri(s"fronts.$domain")
-    )
```

Comment:
I'm not entirely convinced Composer and Fronts need CORS access to Grid either as they proxy though the server IIRC.

Reply:
Update: Fronts and Composer allow you to paste in a URL to a Grid crop and it'll make a request to media-api, so CORS access is needed for these tools.